src/main/java/webserver/http/Cookie.java

```diff
@@ -0,0 +1,37 @@
+package webserver.http;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import static java.util.stream.Collectors.toMap;
+
+@Getter
+@NoArgsConstructor
+@AllArgsConstructor
+@EqualsAndHashCode
+public class Cookie {
+
+    private Map<String, String> entries = new HashMap<>();
```

Comment:
쿠키의 속성 값들은 정해져 있으므로 아래와 같이 선언해도 될 것 같아요 :)
```suggestion
    private final String name;
    private final String value;
    private String domain;
    private String path;
    ...
```